src/components/usa-banner/usa-banner.css.ts

```diff
@@ -46,16 +48,17 @@ export const bannerStyles: CSSResultGroup = [
       --usa-icon-expand-more: url("/src/shared/icons/expand_more.svg");
       --usa-icon-expand-less: url("/src/shared/icons/expand_less.svg");
       --usa-icon-close: url("/src/shared/icons/close.svg");
-      --usa-icon-lock: url("/src/shared/icons/lock.svg");
+      --usa-icon-lock: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='52' height='64' viewBox='0 0 52 64' class='usa-banner__lock-image' role='img' aria-labelledby='banner-lock-description-default' focusable='false'%3E%3Ctitle id='banner-lock-title-default'%3ELock%3C/title%3E%3Cdesc id='banner-lock-description-default'%3ELocked padlock icon%3C/desc%3E%3Cpath fill='%23000000' fill-rule='evenodd' d='M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z'%3E%3C/path%3E%3C/svg%3E");
```

Comment:
Now that the SVG is inline and visible here (great!), I'm noticing that the original SVG has some hard-coded English assistive text in the `<title>` here. I bet we can pull that English out of here in a follow-up PR!

I'm not sure this is the best approach, but here's one way I've done this before:
https://gist.github.com/caseywatts/e6b0600941b1356333f13a6a67e0e285#file-login-gov-component-js-L72-L119

@heymatthenry @ethangardner would it help if I opened up an issue for this?